www/includes/easyparliament/templates/html/mp/votes.php

```diff
@@ -77,16 +77,29 @@
                 <div class="panel">
                     <a name="votes"></a>
                     <h2><?= $full_name ?>&rsquo;s voting in Parliament</h2>
-
-                    <p>
-                    <?= $full_name ?> is a <?= $party ?> MP, and on the <b>vast majority</b> of issues <a href="/voting-information/#party-and-individual-responsibility-for-decisions">follow instructions from their party</a> and vote the <b>same way</b> as other <?= $party ?> MPs.
-                    </p>
+                    <?php $unslugified_comparison_party = ucwords(str_replace('-', ' ', $comparison_party)); ?>
```

Comment:
you don't need this, if you've now set it in mp/index.php?

Reply:
Yep.